.gitmodules

```diff
@@ -1,9 +1,6 @@
 [submodule "engine"]
 	path = engine
-	url = https://github.com/minds/engine-internal.git
```

Comment:
I think this change is significantly reducing your chances of your PR getting approved.